### PR TITLE
[FR] Add documentation for invoices

### PR DESCRIPTION
**Pull request by maririos**
Fixes: https://github.com/Azure/azure-sdk-for-net/issues/15871

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs [hunk 1]
```diff
@@ -34,36 +34,6 @@ public async Task RecognizeInvoicesFromFile()
 
                 foreach (RecognizedForm invoice in invoices)
                 {
-                    FormField invoiceIdField;
-                    if (invoice.Fields.TryGetValue("InvoiceId", out invoiceIdField))
```
Comment (maririos) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample13_RecognizeInvoicesFromFile.cs:38]: Removing some fields just to make it smaller so the Readme doesn't look that big...

File: sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample10_RecognizeInvoices.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Recognize invoices from a URI
+
+To recognize invoices from a URI, use the `StartRecognizeInvoicesFromUriAsync` method. The returned value is a collection of `RecognizedForm` objects -- one for each page in the submitted document.
```
Comment (kristapratico) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample10_RecognizeInvoices.md:22]: > one for each page in the submitted document

not for invoices :(

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample10_RecognizeInvoices.md (hunk 2 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample10_RecognizeInvoices.md]: oh great catch!! Updated

File: sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md [hunk 3]
```diff
@@ -23,6 +23,7 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 - [Recognize custom forms](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample2_RecognizeCustomForms.md)
 - [Recognize receipts](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample3_RecognizeReceipts.md)
 - [Recognize business cards](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample9_RecognizeBusinessCards.md)
+<!--- [Recognize invoices](https://github.com/Azure/azure-sdk-for-net/blob/feature/formrecognizer2.1/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample10_RecognizeInvoices.md)-->
```
Comment (maririos) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md:26]: documented for now so CI doesn't fail when validating the link. I will update in a future PR

**Pull request merged**